composer.json

```diff
@@ -60,7 +60,7 @@
         "symfony/security-csrf": "^4.4 || ^5.3 || ^6.0",
         "symfony/string": "^5.3 || ^6.0",
         "symfony/translation": "^4.4 || ^5.3 || ^6.0",
-        "symfony/translation-contracts": "^1.1.10 || ^2.3",
+        "symfony/translation-contracts": "^1.1.10 || ^2.3 || ^3.0",
```

Comment:
Which package is blocking the upgrade to 3? it is not getting installed.

Reply:
twig/string-extra

Reply:
What should we do? revert this one and wait for string extra? Not sure what changes for this specific contract

Reply:
Looking at the changelog, https://github.com/symfony/contracts/blob/main/CHANGELOG.md#30, I would consider ok to keep it like this. As u want